reoptjl/test/test_job_endpoint.py

```diff
@@ -227,7 +227,7 @@ def test_superset_input_fields(self):
         r = json.loads(resp.content)
         results = r["outputs"]
 
-        self.assertAlmostEqual(results["Financial"]["npv"], 11323.01, places=0)
+        self.assertAlmostEqual(results["Financial"]["npv"], -258533.19, places=-3)
```

Comment:
Did just changing the solver change the result by this much?

Reply:
Ah, no, it's because I had to fix all of the technology sizes to avoid a timeout with the open source solver. Fixing the sizes still tests the capabilities here, which also implies the NPV test is not needed.